bootstrap/bootstrap.sh

```diff
@@ -51,4 +51,10 @@ cd ansible-env/bin
 curl -L https://cpanmin.us -o cpanm
 chmod +x ./cpanm
 
+echo "Installing click, for local python script"
+pip install click
```

Comment:
not sure why this is necessary? (also 👇 )

Reply:
yes but that should be run in a separate Conda environment, right? so the dependency should be installed there.

Reply:
Ok, I'll look into this.